<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>buildContactDlg</class>
 <widget class="QDialog" name="buildContactDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>158</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>备   注  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>111</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>群   组  ：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newContactName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>61</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>62</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>邮   箱  ：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OKBtn">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>210</y>
     <width>80</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newContactNick">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>158</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>210</y>
     <width>80</width>
     <height>34</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newContactGroup">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>添加新联系人</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
